clients/drcachesim/drpt2trace/drpt2trace.cpp

```diff
@@ -74,9 +74,18 @@ static droption_t<std::string>
               "Specifies the file path of the PT raw trace. Please run the "
```

Comment:
Some more details would help in the PR description. Specifically, that the sideband contains events that are relevant only when we have multiple modules, that for kernel traces we expect to have only a single one, and that for user traces we may have multiple if the trace is long enough.

Reply:
Do you mean to add these message to PR's description or in drpt2trace.cpp?

Reply:
In the PR description.

Reply:
Done.

Reply:
So PT does not bother to produce a sideband file if only one module was encountered, right?  That's why passing the module explicitly is required?  I would make that clear.  It would be nicer and more consistent if it would always produce a sideband file.